paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetActivity.kt

```diff
@@ -57,14 +89,13 @@ internal class CustomerSheetActivity : AppCompatActivity() {
 
 @Composable
 private fun Loading() {
-    Column {
-        Text("loading...")
-    }
-}
-
-@Composable
-private fun Data(data: String) {
-    Column {
-        Text(data)
+    val padding = dimensionResource(R.dimen.stripe_paymentsheet_outer_spacing_horizontal)
+    Column(
```

Comment:
This could be just a `Box` and it should take in `modifier` from a method parameter.